### ISSUE TITLE
Users can not highlight read only text in a clause template

### ISSUE THREAD

**Issue by Michael-Grover** [accordproject/web-components#242]
# Bug Report 🐛
Users currently can not highlight read only text in a clause template. A user should be able to highlight read only text, but not be able to change the text. A user may want to highlight read only text so that they can copy and paste it

## Steps to Reproduce

1.  go to the [contract editor demo in the storybook](https://ap-web-components.netlify.app/?path=/story/contract-editor--contract-editor)
2. go to "knobs" and switch on "read only"
3. Try to highlight text in a clause template. it is not possible but should be.
![image](https://user-images.githubusercontent.com/40569793/107574591-dd8d9800-6bbc-11eb-891d-5b53d044fc01.png)

**Comment by K-Kumar-01** [comment]
@Michael-Grover 
I would like to have a go at issue as I am guessing I figured out the problem.

**Comment by K-Kumar-01** [comment]
@Michael-Grover 
I have created a PR for the problem. Let me know if the approach is right or if there are any changes required. :)
